Washing Pole Fix

ThunderDong opened the pull request:
Decreased base physical damage from 121 to 116 and max durability from 20 to 10. This is a ppv1 script so the ParamPatcher needs to be ported to this table before usage.

ThunderDong committed:
Washing Pole Fix

Decreased base physical damage from 121 to 116 and max durability from 20 to 10. This is a ppv1 script so the ParamPatcher needs to be ported to this table before usage.

TheGrandArchives commented:
I would really prefer it if the durability wasn't touched, since it would become nigh useless after future plans to change bonfires.

TheGrandArchives committed:
Rename patch.ce to patch.ct

Ziggylata commented:
Why not just revert it back when that happens?

ThunderDong committed:
Washing Pole Fix 1.1

Removed max durability change

ThunderDong committed:
Changed file name to .ct

Lol

ThunderDong committed:
PPv1 port, Darkmoon Bow Fix

-Ported PPv1 to patch table (can be replaced with ppv2 if desired, but all current scripts will have to be converted)
-When used with Moonlight Arrow, Darkmoon Bow WA now fires arrows that explode on impact and deal magic damage
-Darkmoon Bow int scaling increased from 60 to 70
